live-examples/css-examples/text-decoration/text-underline-offset.html

```diff
@@ -1,28 +1,27 @@
 <section id="example-choice-list" class="example-choice-list" data-property="text-underline-offset">
-<div class="example-choice" initial-choice="true">
-<pre><code class="language-css">text-underline-offset: 3px;</code></pre>
-<button type="button" class="copy hidden" aria-hidden="true">
-    <span class="visually-hidden">Copy to Clipboard</span>
-</button>
-</div>
+	<div class="example-choice" initial-choice="true">
+		<pre><code class="language-css">text-underline-offset: auto;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+			<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
 
-<div class="example-choice">
-<pre><code class="language-css">text-underline-offset: 1rem;</code></pre>
-<button type="button" class="copy hidden" aria-hidden="true">
-    <span class="visually-hidden">Copy to Clipboard</span>
-</button>
-</div>
-<div class="example-choice">
-<pre><code class="language-css">text-underline-offset: from-font;
-font-size: 2rem;</code></pre>
-<button type="button" class="copy hidden" aria-hidden="true">
-    <span class="visually-hidden">Copy to Clipboard</span>
-</button>
-</div>
+	<div class="example-choice">
+		<pre><code class="language-css">text-underline-offset: 8px;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+			<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+	<div class="example-choice">
+		<pre><code class="language-css">text-underline-offset: -0.5rem;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+			<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
 </section>
 
 <div id="output" class="output hidden">
-    <section id="default-example">
-        <p id="example-element">And after all we're only ordinary men</p>
-    </section>
+	<section id="default-example">
+		<p id="example-element">And after all we're only ordinary men</p>
```

Comment:
```suggestion
		<p id="example-element">And after all we are only ordinary</p>
```

This all looks great @NiedziolkaMichal - I realise you didn't introduce this, but in a bid to be more inclusive I'm suggesting removing 'men' :)

Reply:
Of course. Thank you for review.